client/src/utils.js

```diff
@@ -15,14 +18,12 @@ function parseData(parse) {
 	};
 }
 
-const parseDate = timeParse("%Y-%m-%d");
+const parseDate = timeParse("%s");
 
-export function getData() {
-	const promiseMSFT = fetch("https://cdn.rawgit.com/rrag/react-stockcharts/master/docs/data/MSFT.tsv")
-		.then(response => response.text())
-		.then(data => tsvParse(data, parseData(parseDate)))
-	return promiseMSFT;
+const apiEndpoint = apiUrl + "/chart";
 
+export function getData(begin, end, ticker, resolution) {
+	return http.get(apiEndpoint + "?from=" + begin + "&to=" + end + "&ticker=" + ticker + "&resolution=" + resolution);
```

Comment:
You need to follow our conventions and define this in a `....service.js` file. 

This should be in a new file called `services/chartingService.js`, not in utils.

Reply:
Done